streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -3173,12 +3173,13 @@ void to(final TopicNameExtractor<K, V> topicExtractor,
      * (cf. {@link #transformValues(ValueTransformerSupplier, String...) transformValues()} )
      * <p>
      * Note that it is possible to emit multiple records for each input record by using
-     * {@link ProcessorContext#forward(Object, Object) context#forward()} in
+     * {@link org.apache.kafka.streams.processor.ProcessorContext#forward(Object, Object) context#forward()} in
```

Comment:
Transformation will be migrated separately via https://issues.apache.org/jira/browse/KAFKA-10603, so we just need to make sure the javadoc references the correct context. Note that for this reason, we did _not_ deprecate the old ProcessorContext.